lib/rubocop/cop/lint/literal_in_interpolation.rb

```diff
@@ -100,6 +102,40 @@ def autocorrected_value_for_array(node)
           contents_range(node).source.split.to_s.gsub('"', '\"')
         end
 
+        def autocorrected_value_for_hash(node)
+          hash_string = node.children.map do |child|
+            key = autocorrected_value_in_hash(child.key)
+            value = autocorrected_value_in_hash(child.value)
+            "#{key}=>#{value}"
+          end.join(', ')
+
+          "{#{hash_string}}"
+        end
+
+        def autocorrected_value_in_hash(node)
+          case node.type
+          when :int
+            node.children.last.to_i.to_s
+          when :float
+            node.children.last.to_f.to_s
+          when :str
+            '\"' + node.value.to_s.gsub('"') { '\\\\\"' } + '\"'
+          when :sym
+            # TODO: We need to detect symbol unacceptable names more correctly
+            if / |"|'/ =~ node.value.to_s
```

Comment:
A gem [unparser](https://github.com/mbj/unparser) provides more simple (and better) way to generate code strings for autocorrect result. 
Should we add it to the list of dependencies?
```suggestion
            Unparser.unparse(node).gsub('"') { '\"' }
```